awx/api/views/__init__.py

```diff
@@ -364,11 +364,14 @@ class InstanceList(ListAPIView):
     serializer_class = serializers.InstanceSerializer
     search_fields = ('hostname',)
 
+    def get_queryset(self):
+        return super().get_queryset().exclude(node_type='hop')
+
 
 class InstanceDetail(RetrieveUpdateAPIView):
 
     name = _("Instance Detail")
-    model = models.Instance
+    queryset = models.Instance.objects.exclude(node_type='hop')
```

Comment:
I think I agree with Alan here. Or at least share the confusion. Why do we want to exclude hop nodes?

Reply:
Recording this here for posterity, but the exclusions of hop nodes I've added are purely for existing endpoints so that users don't get confused by seeing additional hosts that will never be part of their job execution capacity.  After offline discussion, it seemed like you were both fine with this, since the endpoint brought in with #11454 _will_ include the hop nodes.